pages/[user]/[type].tsx

```diff
@@ -44,7 +47,7 @@ export default function Type(props): Type {
     setTimeFormat(is24hClock ? "HH:mm" : "h:mma");
   };
 
-  return (
+  return isReady && (
```

Comment:
Codacy has a fix for the issue: [Insert `(⏎····`](https://app.codacy.com/gh/calendso/calendso/pullRequest?prid=7734611)
```suggestion
  return (
```